src/main/java/org/repodriller/filter/diff/OnlyDiffsWithoutFileTypes.java

```diff
@@ -0,0 +1,23 @@
+package org.repodriller.filter.diff;
```

Comment:
This class violates the DRY principle with respect to the logic of "does this path have this extension".
Can you perhaps introduce a static function in RDFileUtils for this, and then call it from both classes?

Reply:
Yes.